miden/src/tools/mod.rs

```diff
@@ -34,32 +34,31 @@ impl Analyze {
         let stack_inputs = input_data.parse_stack_inputs()?;
         let advice_provider = input_data.parse_advice_provider()?;
 
-        let program_info: ProgramInfo = analyze(program.as_str(), stack_inputs, advice_provider)
-            .expect("Could not retrieve program info");
+        let execution_details: ExecutionDetails =
+            analyze(program.as_str(), stack_inputs, advice_provider)
+                .expect("Could not retrieve program info");
 
-        println!("{}", program_info);
+        println!("{}", execution_details);
 
         Ok(())
     }
 }
 
-// PROGRAM INFO
+// EXECUTION DETAILS
 // ================================================================================================
 
-/// Contains info of a program. Used for program analysis. Contains the following fields:
-/// - total_vm_cycles: vm cycles it takes to execute the entire program
-/// - total_noops: total noops executed as part of a program
-/// - asm_op_stats: vector of [AsmOpStats] that contains assembly instructions and
-///   the number of vm cycles it takes to execute the instruction and the number of times the
-///   instruction is run as part of the given program.
+/// Contains details of the exeuction of a program, used for program analysis
```

Comment:
minor nit: I'd change this to "Contains details of executing a program, used for program analysis."